features/invitations/invite_request.feature

```diff
@@ -87,14 +87,51 @@ Feature: Invite requests
       And I should see how long I have to activate my account
       And I should see "If you haven't received this email within 24 hours"
 
+  Scenario: A user can't send an invite with an empty email
```

Comment:
We can keep one of the new scenarios to test the error message "Please enter an email address." The other scenarios are better as specs, though.

Reply:
The "A user can't update an invalid invite with an empty email" one might not translate well into a spec... it's like
- submit the form on users/name/invitations with invalid info, 
- the invitation show view will render as a result
- fill in the form on the rendered show view incorrectly and submit that as well